src/components/Coaching/CoachingRow/CoachingRow.tsx

```diff
@@ -0,0 +1,141 @@
+import { Box, styled, Typography } from '@material-ui/core';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { CoachFragment } from '../LoadCoachingList.generated';
+import { currencyFormat, percentageFormat } from 'src/lib/intlFormat';
+import StyledProgress from 'src/components/StyledProgress';
+
+interface Props {
+  coach: CoachFragment;
+}
+
+const CoachingRowWrapper = styled(Box)(({ theme }) => ({
+  width: '100%',
+  maxWidth: '950px',
+  margin: 'auto',
+  padding: theme.spacing(1),
+}));
+
+const CoachingNameText = styled(Typography)(({ theme }) => ({
+  margin: theme.spacing(2),
+}));
+
+const CoachingProgressLabelContainer = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  justifyContent: 'space-between',
+  margin: theme.spacing(2, 2, 0),
+}));
+
+export const CoachingRow: React.FC<Props> = ({ coach }) => {
+  const { t } = useTranslation();
+
+  const {
+    monthlyGoal,
+    currency,
+    name,
+    totalPledges,
+    receivedPledges,
+    primaryAppeal,
+  } = coach;
+
+  const calculatedMonthlyGoal = monthlyGoal ? monthlyGoal : 0;
+  const percentageRecievedPledges = receivedPledges / calculatedMonthlyGoal;
+  const percentageTotalPledges = totalPledges / calculatedMonthlyGoal;
+  const recievedCurrency = currencyFormat(receivedPledges, currency)
+    ? currencyFormat(receivedPledges, currency)
```

Comment:
Is the check here needed? I think ```currencyFormat ```with always at least return ```0``` and the currency symbol.